### DISCUSSION TITLE
Personnalisation du message alertBigPrivacy

### DISCUSSION THREAD

**Discussion by mikedevdur** [AmauriC/tarteaucitron.js#1100] [Q&A] [answered]
J'ai vu que l'on pouvait personnaliser le message pour les mesures d'audience de cette façon
`<script>
var tarteaucitronCustomText = {
    "analytic": {
        "details": "VOTRE DESCRIPTION ANALYTIC"
    },
    "social": {
        "details": "VOTRE DESCRIPTION RESEAUX SOCIAUX"
    },
};
</script>`

J'aimerais pouvoir personnaliser alertBigPrivacy (par exemple le texte serait disponible dans une variable php). J'ai essayé avec la solution proposée ci-dessus ca ne fonctionne pas . Est il possible de customiser alertBigPrivacy sans modifier le fichier de langue ?

Je vous remercie d'avance pour la réponse

**Comment by AmauriC** [comment, marked as answer]
Tout est modifiable de la même manière:

```
<script>
var tarteaucitronCustomText = {
   "alertBigPrivacy": "TEXTE PRINCIPALE DE LA BANNIERE",
   "analytic": { "details": "VOTRE DESCRIPTION ANALYTIC" },
   "social": { "details": "VOTRE DESCRIPTION RESEAUX SOCIAUX" },
 }; </script>
```

**Comment by mikedevdur** [comment]
Parfait ca fonctionne. Merci beaucoup
